src/legacy/js/functions/_t4BulletinEditor.js

```diff
@@ -101,6 +101,10 @@ function bulletinEditor(collectionId, data) {
         data.description.nationalStatistic = $("#metadata-list input[type='checkbox']").prop('checked') ? true : false;
     });
 
+    $("#survey-checkbox").prop('checked', data.description.survey ? true : false).click(function () {
```

Comment:
No need for the ternary operator here?
```suggestion
    $("#survey-checkbox").prop('checked', data.description.survey).click(function () {
```

Also should this prop be included in articles and releases too? It looks like we do them in the handlebars template rather than js for them. We should try and be consistent. Which one would be a better practice?

Reply:
If we don't have the ternary won't it just try and inject the value of .survey into the checked property when it only accepts true or false? 

So if something has the census brand attached to it won't it try and inject the string "census" and if it does not have the census brand won't it try and inject null or possible undefined?

In terms of the prop being included for articles and releases I actually tried to remain consistent by having the bulletin and release pages handle the checking in the handlebars instead but they refused, why I have no idea as I could see description.survey in the template data when it passed through.

I could change articles and releases to instead use the prop but my thinking was that each file remains consistent with itself which I think is less confusing. 

For example if I change articles and releases to have the changes bulletins and datasets have they will then have the national statistic checkbox property decided in the handlebars but the census branding checkbox decided by the prop which I think in it's own way is more confusing.

Reply:
You are absolutely right! Not sure why I thought `data.description.survey` contained a boolean... Yes, the ternary operator is needed, sorry for the confusion!

Re the checked props, it all seems odd! I have been playing with it and I can't get the articles or releases to work as is. Not sure what is different to the national statistics checkbox that seem to work. Maybe we should go to the bottom of this? 
I feel like we should try and make it consistent across files (work with all checkbox in the same way) even if that means a file might start having inconsistencies with itself? But open to discuss it

Reply:
Just noticed this changes would make the release file not be consistent with itself either! 🤷

Reply:
Made the handling of checkboxes consistent across files (https://github.com/ONSdigital/florence/pull/773)